lms/templates/admin/instance.new.html.jinja2

```diff
@@ -2,25 +2,27 @@
 
 {% extends "lms:templates/admin/base.html.jinja2" %}
 
-
 {% block header %}
 New application instance
 {% endblock %}
 
 {% block content %}
 
-<form method="POST" action="{{ request.route_url("admin.registration.new.instance", id_=lti_registration.id) }}">
-    <input type="hidden" name="csrf_token" value="{{get_csrf_token()}}">
-    <fieldset class="box mt-6">
-        <legend class="label has-text-centered">Registration</legend>
+<fieldset class="box mt-6">
+    <legend class="label has-text-centered">Registration</legend>
 
-        {{ macros.registration_preview(request, lti_registration) }}
-    </fieldset>
+    {{ macros.registration_preview(request, lti_registration) }}
+</fieldset>
```

Comment:
I've moved this above the form, because it's not part of it, and it more closely maps to the visual layout.